BackofficeBundle/Resources/public/coffee/table/tableHeader/TableviewSearchHeader.coffee

```diff
@@ -0,0 +1,27 @@
+TableviewSearchHeader = OrchestraView.extend(
+
+  tagName: "tr"
+  className: "header-search-input"
+
+  initialize: (options) ->
+    @options = options
+    @render()
+    widgetChannel.trigger 'ready', @
+    return
+
+  render: ->
+    for type, column in @options.inputHeader
+        type = "empty" if not tableFieldViewConfigurator[type]?
+        viewClass = tableFieldViewConfigurator[type]
```

Comment:
i test set the configuration when jquery is loaderd but it's not work beacuse the render method will be launched before the set configuration of other file ( like BooleanFieldSearchView or TextFieldSearchView)